libkbfs/init.go

```diff
@@ -94,6 +98,18 @@ func GetDefaultMDServer(ctx Context) string {
 	}
 }
 
+// GetDefaultMetadataVersion returns the default metadata version per run mode.
+func GetDefaultMetadataVersion(ctx Context) MetadataVer {
+	switch ctx.GetRunMode() {
+	case libkb.StagingRunMode:
+		return SegregatedKeyBundlesVer
+	case libkb.ProductionRunMode:
+		return InitialExtraMetadataVer
+	default:
+		return InitialExtraMetadataVer
```

Comment:
Given that we're preparing for it to be in prod, I'd say that it might be a good idea to enable kbfsdocker for md v3